executor/index_lookup_hash_join.go

```diff
@@ -324,12 +329,10 @@ func (iw *indexHashJoinInnerWorker) getNewJoinResult(ctx context.Context) (*inde
 	return joinResult, ok
 }
 
-func (iw *indexHashJoinInnerWorker) buildHashTableForOuterResult(ctx context.Context, cancelFunc context.CancelFunc, task *indexHashJoinTask) {
-	var (
-		keyBuf, valBuf  = make([]byte, 0, 64), make([]byte, 8)
-		rowIdx, numRows = 0, task.outerResult.NumRows()
-		err             error
-	)
+func (iw *indexHashJoinInnerWorker) buildHashTableForOuterResult(ctx context.Context, cancelFunc context.CancelFunc, task *indexHashJoinTask, h hash.Hash64) {
+	rowIdx, numRows := 0, task.outerResult.NumRows()
+	buf := make([]byte, 1)
+	task.lookupMap = newRowHashMap()
```

Comment:
`newRowHashMap` is added an argument here, https://github.com/pingcap/tidb/pull/11937/files#diff-4f7e05240341383641662778116ddf13R236
We should merge #11937, then fix it here.